exchanges/binance/binance_test.go

```diff
@@ -661,16 +662,32 @@ func TestGetIndexAndMarkPrice(t *testing.T) {
 
 func TestGetFuturesKlineData(t *testing.T) {
 	t.Parallel()
-	_, err := b.GetFuturesKlineData(t.Context(), currency.NewPairWithDelimiter("BTCUSD", "PERP", "_"), "1M", 5, time.Time{}, time.Time{})
-	if err != nil {
-		t.Error(err)
+	r, err := b.GetFuturesKlineData(t.Context(), currency.NewPairWithDelimiter("BTCUSD", "PERP", "_"), "1M", 5, time.Time{}, time.Time{})
+	require.NoError(t, err, "GetFuturesKlineData must not error")
+	if mockTests {
+		require.Len(t, r, 5, "GetFuturesKlineData must return 5 items in mock test")
```

Comment:
🦍 👑  Reckon this will be painful when it fails. Recommend
```suggestion
		require.Equal(t, 5, len(r), "GetFuturesKlineData must return 5 items in mock test")
```